CHANGELOG.md

```diff
@@ -30,6 +48,7 @@ This release contains contributions from (in alphabetical order):
 ### Contributors ✍️
 
 This release contains contributions from (in alphabetical order):
+Austin Huang
 
 Utkarsh Azad
 Mashhood Khan
```

Comment:
```suggestion
Utkarsh Azad
Lillian M. A. Frederiksen
Austin Huang
Mashhood Khan
```

Reply:
Probably messed this up when resolving a merge conflict 😅